packages/ses/src/intrinsics.js

```diff
@@ -166,6 +182,7 @@ export const makeIntrinsicsCollector = () => {
  * @param {object} globalObject
  */
 export const getGlobalIntrinsics = globalObject => {
+  // TODO pass a proper reporter to `makeIntrinsicsCollector`
```

Comment:
Threading `reporter` through here does seem appropriate to me. A way to avoid it would be to inject the intrinsics collector instead, and construct that in `lockdown` in the presence of the reporter.

Reply:
What about the call to `getGlobalIntrinsics` in compartment-shim.js? That's the problematic one.

Reply:
Done this much. But moved an equally broken `console` reference into compartment.shim.js' call to `getGlobalIntrinsics`, so still unsolved.

Reply:
I think it might be sufficient to default `reporter` to `muteReporter` in `getIntrinsics`, because I assume we do not want to warn every time a `Compartment` gets constructed, and only when constructing intrinsics in service to `lockdown`. Does that sound right?

Reply:
> Does that sound right?

Yes. Done. Thanks!